.github/utils/CheckExamples/CMakeLists.txt

```
@@ -0,0 +1,45 @@
+# This just checks that files in the Check folders are compilable to avoid
+# committing incorrect examples. It's done here, separately and in bulk, so that
+# no overhead is added by default when submitting a new check. You'll hopefully
+# only have to know this exists if your example fails to compile
+
+cmake_minimum_required(VERSION 3.25)
+project("Test that the check examples compile")
+
+enable_language(Fortran OPTIONAL)
```

We can always ensure that the runner image has a Fortran compiler, so we wouldn't really need to make this optional. Moreover, it appears that the keyword doesn't mean anything as of now: https://cmake.org/cmake/help/latest/command/enable_language.html

```suggestion
enable_language(Fortran)
```

---

done